doc/tutorials/content/function_filter.rst

```diff
@@ -0,0 +1,79 @@
+.. _conditional_removal:
+
+Removing outliers using a custom non-destructive condition
+----------------------------------------------------------
+
+This document demonstrates how to use the FunctionFilter class to remove points from a PointCloud that do not satisfy a custom criteria. This is a cleaner
```

Comment:
In most places it is referred to as `FunctorFilter`, not `FunctionFilter`. Should be consistent to avoid confusion

Reply:
The issue with FunctorFilter is that the lambda type is part of the ctor and the template and it's not straightforward to use it. Feel free to iterate on the design though (part of reason of putting it in experimental is that we can break API as and when needed)